ansible/group_vars/all

```diff
@@ -46,27 +55,14 @@ nginx_vhosts:
         auth_basic_user_file /usr/local/biblebox/etc/basicauth;
       }
 
-      location /shared/ {
```

Comment:
I'm not sure about the desire to quickly remove/hide content. I haven't heard about that use-case, but it'd be good to ask